### ISSUE TITLE
Dropdown react controlled behavior - internal value doesn't reset

### ISSUE THREAD

**Issue by MarekLacoAXA** [axa-ch/patterns-library#1579]
Unsure if related to #1560 or not.

Version: 4.0.1 (prevous version 3 did also have same problem).

IS:
In react, when Dropdown receives a value, it properly renders the received-value option. But, somehow it still remembers its previous state, which leads to this problem: the previous option **can not** be selected by clicking on it again.

SHOULD:
When Dropdown receives a value (in our case it is "reset to Show All", then pre prevous-selected option can be freely selected again).

Step 1:
![image](https://user-images.githubusercontent.com/43809369/71673802-01579b00-2d7a-11ea-8b5a-8d6ca072150f.png)

Step 2:
![image](https://user-images.githubusercontent.com/43809369/71673814-074d7c00-2d7a-11ea-8b3f-ccdd78f012c9.png)

**Comment by danyball** [comment]
Hi Marek, we will check that next week.

**Comment by MarekLacoAXA** [comment]
Finally I was able to reproduce this problem:

Issue happens only under these conditions:
- `createAXADatepicker` is imported BEFORE `createAXADropdown` gets imported
- using `@axa-ch/datepicker"` version 3

After upgrading to newest `"@axa-ch/datepicker": "^4.0.1", this issue with AXADropdown is gone.

Was really difficult to reproduce, because I was expecting all kind of different causes than this one.

Closing.
